draft-ietf-ohai-ohttp.md

```diff
@@ -226,7 +235,7 @@ Oblivious HTTP is more costly than a direct connection to a server.  Some costs,
 like those involved with connection setup, can be amortized, but there are
 several ways in which oblivious HTTP is more expensive than a direct request:
 
-* Each oblivious request requires at least two regular HTTP requests, which adds
+* Each Oblivious Gateway requires at least two regular HTTP requests, which adds
```

Comment:
is this correct? My understanding of the old text was comparing an "end-to-end request" (a request from client to target) with just a normal HTTP request operation. 

Maybe I was wrong before and the intent was to talk about the Gateway. But saying the Gateway requires two requests doesn't seem seem right either.